### PR TITLE
cleaning up debug label names

### PR DESCRIPTION
**Pull request by rahwang**
Tweaking some names of debug label options added in [#5188](https://github.com/AnalyticalGraphicsInc/cesium/pull/5188) as per @pjcozzi 's comments

### CODE REVIEW

File: Source/Scene/Cesium3DTile.js [hunk 1]
```diff
@@ -484,6 +484,21 @@ define([
...
+         *
+         * @readonly
+         *
+         * @private
+         */
+        commandsLength : {
+            get : function() {
+                if (defined(this._commandsLength)) {
+                    return this._commandsLength;
+                }
```
Comment (lilleyse) [inline Source/Scene/Cesium3DTile.js]: Just `return this._commandsLength;` is fine since even with the if block `undefined` would be returned.

Comment (lilleyse) [inline Source/Scene/Cesium3DTile.js]: Actually probably the better thing to do is return 0 if not defined.

Comment (lilleyse) [inline Source/Scene/Cesium3DTile.js]: Oh it is already set to 0 initially, so just returning `this._commandsLength` is good after all.

**Comment by rahwang** [comment]
Fixed @lilleyse

**Comment by pjcozzi** [comment]
LGTM, thanks @rahwang!

@lilleyse please merge when ready.

File: Source/Scene/Cesium3DTile.js [hunk 2]
```diff
@@ -526,6 +526,19 @@ define([
...
+        },
+
+        /**
+         * Returns the number of draw commands used by this tile.
+         *
+         * @readonly
+         *
+         * @private
+         */
+        commandsLength : {
```
Comment (lilleyse) [inline Source/Scene/Cesium3DTile.js]: There are a couple places still referencing `._commandsLength` from outside of this class.

Comment (rahwang) [inline Source/Scene/Cesium3DTile.js]: Thanks for catching that @lilleyse . That should be all of them!

**Pull request merged**